L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.h

```diff
@@ -17,22 +17,22 @@
 
 
 // Class declaration
-class L1TMuonEndCapTrackProducer : public edm::EDProducer {
+class L1TMuonEndCapTrackProducer : public edm::stream::EDProducer<> {
 public:
   explicit L1TMuonEndCapTrackProducer(const edm::ParameterSet&);
   ~L1TMuonEndCapTrackProducer() override;
 
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
 private:
-  void beginJob() override;
   void produce(edm::Event&, const edm::EventSetup&) override;
-  void endJob() override;
 
-  //virtual void beginRun(edm::Run const&, edm::EventSetup const&);
-  //virtual void endRun(edm::Run const&, edm::EventSetup const&);
-  //virtual void beginLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const&);
-  //virtual void endLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const&);
+  //void beginJob() override;
```

Comment:
why having all these declarations commented?

Reply:
Because we're not doing anything in these blocks at the moment, but we might in the future.  It's helpful to see how to re-add them if necessary, also for debugging, e.g. adding cout statements in beginJob().

Reply:
@abrinke1 well, this can make the code more messy, as it is easy to decomment and then forget. Anyway...